scripts/export-user-data.ts

```diff
@@ -0,0 +1,68 @@
+import fs from 'fs';
+import { semestersCollection } from './firebase-config';
+
+class Rating {
+  userId: number;
+
+  courseId: number;
+
+  rating: number;
+
+  timestamp: number;
+
+  constructor(id: number, course: number, rating: number, timestamp: number) {
+    this.userId = id;
+    this.courseId = course;
+    this.rating = rating;
+    this.timestamp = timestamp;
+  }
```

Comment:
I think having 4 unnamed numeric arguments is confusing. Could we pass in an object instead?